wiki/markdown/running-rules-on-objects.md

```diff
@@ -1,11 +1,21 @@
 # Running Rules On Objects
 
-|Method|Endpoint             |
-|------|---------------------|
-|POST |/rules/rule-check|
-|POST | /rules/rule-check/aggregation|
-|GET | /rules/rule-check/testRulePageEnabled |
-
+|Method|Endpoint                               |
+|------|---------------------------------------|
+|GET   | /rules                                |
+|POST  | /rules/rule-check                     |
+|POST  | /rules/rule-check/aggregation         |
+|GET   | /rules/rule-check/testRulePageEnabled |
+
+## Checking the active rules in your backend instance
+
+You can fetch the rules content from the backend to see what is currently running.
```

Comment:
--> "to see what rules Eiffel Intelligence is currently using".

Reply:
Done.